README.md

```diff
@@ -1,34 +1,67 @@
-# Chime SDK UI Component Library
+# Chime SDK React Components Library
```

Comment:
Should we start with Amazon here as we do in the JS SDK README?

Reply:
yeah good point